### PR TITLE
Support managementGroup and subscription scopes in Blueprint API version 2018-11-01

### PR DESCRIPTION
**Pull request by filizt**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/blueprint/resource-manager/Microsoft.Blueprint/preview/2018-11-01-preview/blueprintDefinition.json [hunk 1]
```diff
@@ -570,33 +636,6 @@
           "nextLinkName": "nextLink"
         }
       }
-    },
```
Comment (ravbhatnagar) [inline specification/blueprint/resource-manager/Microsoft.Blueprint/preview/2018-11-01-preview/blueprintDefinition.json]: why is the operations API been removed?

**Review by ravbhatnagar — COMMENTED** [review]
looks fine, just one note.

**Comment by ravbhatnagar** [comment]
@filizt - Why is the operations API linter rule suppressed? Your RP should still expose the available operations

**Comment by haitch** [comment]
@ravbhatnagar it doesn't make sense to expose operations in swagger.  Swagger also explains what operations this RP supports.

**Comment by haitch** [comment]
@ravbhatnagar because we are extension resource, the url path include /providers/Microsoft.Management, then the linter complain about multiple resource providers.

**Comment by ravbhatnagar** [comment]
confirmed with @haitch and the operations API is present. The suppression is needed due to the multiple provider segments and linter throwing an issue for that.

**Comment by jhendrixMSFT** [comment]
@AutorestCI regenerate azure-sdk-for-go

**Review by praries880 — APPROVED** [review]

**Pull request merged**